.vscode/settings.json

```diff
@@ -1,4 +1,9 @@
 {
     "python.linting.pylintEnabled": true,
-    "python.linting.enabled": false
+    "python.linting.enabled": false,
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#3D2732",
+        "titleBar.activeBackground": "#553746",
+        "titleBar.activeForeground": "#FCFAFB"
+    }
```

Comment:
can you not commit these to git and have them as your local preferences?
otherwise everyone will be forced to have their vscode with these personal preferences